code/script.js

```diff
@@ -1,13 +1,20 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper') 
+const form = document.getElementById('name-form')
+const inputValue = document.getElementById('input-name')
 
 // Global variables, if you need any, declared here
+let currentQuestion = 0
+let yourName = ""
+let continent = ""
```

Comment:
Nice use of globals so that you can use them in multiple functions without having to continue to pass them down